common_logging/src/main/java/uk/gov/dhsc/htbhf/logging/event/Event.java

```diff
@@ -1,20 +1,19 @@
-package uk.gov.dhsc.htbhf.logging;
+package uk.gov.dhsc.htbhf.logging.event;
 
 import com.fasterxml.jackson.annotation.JsonAnyGetter;
-import lombok.Builder;
 import lombok.Data;
+import uk.gov.dhsc.htbhf.logging.EventType;
 
 import java.time.LocalDateTime;
 import java.util.Collections;
 import java.util.Map;
 import java.util.TreeMap;
 
 @Data
-@Builder
```

Comment:
Why did you remove `@Builder`?

Reply:
Because it prevented me from putting a builder in the subclass. I figured that the constructor only had 3 parameters, all different types, so we didn't need a builder. I'm also thinking that creating subclasses of Event seems like a nice way to go.

Reply:
Ok, if we're definitely going down the sub-class route, then should this be abstract? It does solidify the intent then.

Reply:
I hadn't intended to force the subclass to be the only way of doing this, but maybe we should talk about it and see what the general feeling is.

Reply:
At the moment we don't know what the general usage is as we only have two of them, but it feels like making this the equivalent of a `BaseEvent` like the BaseEntity might be good, it doesn't feel like we'd have generic events, they'll always be specific. Maybe it's too early to tell, I don't have strong feelings either way.